packages/server-wallet/src/version.ts

```diff
@@ -0,0 +1 @@
+export const WALLET_VERSION = ''; // Will be overwritten during npm's 'prepare' lifecycle script
```

Comment:
Isolated to a single file to reduce risk of corrupting other bits of code.

Reply:
Did you consider using `read-pkg` to read the version (asynchronously)?

Since we now create wallets through a [static `create` function](https://github.com/statechannels/statechannels/blob/master/packages/server-wallet/src/wallet/index.ts#L16), we can make it asynchronous and inject the wallet version there.